### DISCUSSION TITLE
clearNodeTransform with animations

### DISCUSSION THREAD

**Discussion by kzhsw** [donmccurdy/glTF-Transform#1008] [General]
Currently the function [clearNodeTransform](https://github.com/donmccurdy/glTF-Transform/blob/1203855141862134f49106c30382238778ee5a24/packages/functions/src/clear-node-transform.ts) could apply local transform to mesh primitives.
It seems morph targets are already [supported](https://github.com/donmccurdy/glTF-Transform/blob/1203855141862134f49106c30382238778ee5a24/packages/functions/src/transform-mesh.ts#L59), but could there be an option or function to apply this on `translation`, `rotation`, `scale` [animations](https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/Specification.adoc#animations)?

**Comment by donmccurdy** [comment]
@kzhsw I think that'd be a useful addition, yes – just opened a feature request ticket for it in https://github.com/donmccurdy/glTF-Transform/issues/1013. Not something that I'm planning to work on right now, but would be welcome if you would like to give it a shot!

**Comment by Ruth678** [comment]
Hi, 
I hope you all are doing well. I like this information and it is very helpful to me. If anyone has a question about this please feel free to ask me.
